### PR TITLE
fix ua_utils.data_type_to_variant_type for enum types

### PR DESCRIPTION
**Pull request by bitkeeper**
See #426 about data_type_to_variant_type .

### CODE REVIEW

File: opcua/common/ua_utils.py [hunk 1]
```diff
@@ -203,16 +203,8 @@ def data_type_to_variant_type(dtype_node):
...
-        bnames = [d.BrowseName.Name for d in descs]
-        if "EnumStrings" in bnames:
-            return ua.VariantType.LocalizedText
-        elif "EnumValues" in bnames:
-            return ua.VariantType.ExtensionObject
-        else:
-            raise ua.UaError("Enumeration must have a child node describing its type and values")
-
+        # we have an enumeration, value is a Int32
+        return ua.VariantType.Int32
```
Comment (oroulet) [inline opcua/common/ua_utils.py]: from what you wrote today, this is correct. I am curious to see the consequences of that change...

Comment (bitkeeper) [inline opcua/common/ua_utils.py]: I think within the project itself not much, because I couldn't find any actual use of that code except in the tests.

**Comment by oroulet** [comment]
It is probably the modeler and client GUI that use that method. Maybe the
command line too

**Comment by zerox1212** [comment]
Please test this change with modeler. If I remember correctly modeler uses it when you edit attributes of a variable or property node. There is a good chance you will find this method used in the widgets library.

**Comment by oroulet** [comment]
I am looking at modeler/client

**Comment by oroulet** [comment]
Seems to fix some bugs in modeller, could not get any crash . Thanks the analysis and patch @bitkeeper

**Pull request merged**